webapp/src/store/boards.ts

```diff
@@ -81,7 +81,9 @@ export const updateMembersEnsuringBoardsAndUsers = createAsyncThunk(
             }
             const user = await client.getUser(m.userId)
             if (user) {
-                thunkAPI.dispatch(addBoardUsers([user]))
+                const deleted = !m.schemeAdmin && !m.schemeEditor && !m.schemeViewer && !m.schemeCommenter
+                const processor = deleted ? removeBoardUsers : addBoardUsers
```

Comment:
Personal opinion: This looks like DRY, but for me it is just less readable. For me is way cleaner to have something like:
```ts
if (delete) {
    thunkAPI.dispatch(removeBoardUsers([user]))
} else {
    thunkAPI.dispatch(adBoardUsers([user]))
}
```

Reply:
Done.